specificatie/hypotheken.yaml

```diff
@@ -9,7 +9,7 @@ paths:
     get:
       operationId: GetHypothekenKadastraalOnroerendeZaak
       description: |
-        Het raadplegen van hypotheken die rusten op een kadastraal onroerende zaak met bijbehorende hypotheekhouder(s). Een hypotheekhouder vestigt als geldverstrekker een recht van hypotheek als zekerheid voor de lening.
+        Raadpleeg hypotheken die rusten op een kadastraal onroerende zaak en bijbehorende hypotheekhouder(s). Een hypotheekhouder vestigt als geldverstrekker een recht van hypotheek als zekerheid voor de lening.
```

Comment:
Elke zin in een eigen regel? Dit gebeurt soms wel en soms niet. Is wel makkelijker bij het lezen. Dan hoeft je niet heen en weer te scrollen